scientific/src/commonTest/kotlin/unit/TimeUnitTest.kt

```diff
@@ -163,4 +164,27 @@ class TimeUnitTest {
     fun timeFromWeightAndMassFlowRateTest() {
         assertEqualScientificValue(1(Hour), 2(Gram) / 2(Gram per Hour))
     }
+
+    @Test
+    fun timeFromKinematicViscosityAndAreaTest() {
+        assertEqualScientificValue(1(Second), (2(SquareMeter) / 2(SquareMeter per Second)))
+        assertEqualScientificValue(
+            1(Second),
+            (2(SquareMeter).convert(SquareCentimeter) / 2(SquareMeter per Second))
+        )
+        assertEqualScientificValue(
+            1(Second),
+            (2(SquareMeter).convert(SquareFoot) / 2(SquareMeter per Second))
```

Comment:
I'm having a difficulty to understand why you do `2(SquareMeter).convert(SquareFoot)` here?
Does it test something related to this PR apart from the fact you want to have exact numerical value of `2m^2` expressed in square foots?
Again nothing wrong but quite confusing without some addition info

Reply:
Honestly, now that i see it i don't see the point of this and even if this is correct.